tests/test_computation.py

```diff
@@ -40,33 +40,35 @@ def nested_computation(state: State, args: list[torch.Tensor]):
     out_9 = state.correlation(y, z)
     out_10 = state.linear_regression(x, y)
     slope, intercept = out_10[0][0][0], out_10[0][1][0]
-    reshaped = torch.tensor([
-        out_0,
-        out_1,
-        out_2,
-        out_3,
-        out_4,
-        out_5,
-        out_6,
-        out_7,
-        out_8,
-        out_9,
-        slope,
-        intercept,
-    ]).reshape(1,-1,1)
+    reshaped = torch.cat((
+        out_0.unsqueeze(0),
+        out_1.unsqueeze(0),
+        out_2.unsqueeze(0),
+        out_3.unsqueeze(0),
+        out_4.unsqueeze(0),
+        out_5.unsqueeze(0),
+        out_6.unsqueeze(0),
+        out_7.unsqueeze(0),
+        out_8.unsqueeze(0),
+        out_9.unsqueeze(0),
+        slope.unsqueeze(0),
+        intercept.unsqueeze(0),
+    )).reshape(1,-1,1)
```

Comment:
I think let's not do state.cat just for concatenating scalar value... since people should be able concatenate tensor array, etc. as well

and in case of concatenate diffferent-dimensions tensor arrays, they need to kind of manipulate the shape of those tensors to match as well, which will result in state.cat taking more inputs to do those reshape stuffs... so will not really simplify stuffs compared to torch.cat anyway